ci/Jenkinsfile.template

```diff
@@ -195,11 +195,11 @@ pipeline {
                     }
                 }
                 }
-                retryWithTimeout(1000 /* seconds */, 5 /* retries */) {
-                    script {
-                        h2o4gpuUtils.publishToPoboys("${platform}")
-                    }
-                }
+                //retryWithTimeout(1000 /* seconds */, 5 /* retries */) {
```

Comment:
why not needed?  I thought required for conda?  @hemenkapadia

Reply:
Yes it is needed IMHO. We have an internal anaconda server where we upload conda packages for our builds. http://poboys.h2o.ai:6969/poboys/pkgs/linux-64  contains h2o4gpu builds. It allows one click relase to anaconda.org once we finalize a build version to release.

Reply:
Looks this part was removed manually from every config file, so I've decided to remove(comment out actually) it from the template. I'll add it back and regenerate configurations.